Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -8746,6 +8746,59 @@ class UnparsedCalculationNode final : public CalculationNode {
     ComponentValue m_component_value;
 };
 
+// https://html.spec.whatwg.org/multipage/images.html#parse-a-srcset-attribute
```

Comment:
I was wondering what srcset had to do with CSS. Turns out this is just the wrong spec link, should be <https://html.spec.whatwg.org/multipage/images.html#parsing-a-sizes-attribute>. Bad Kling! No cookie!